czech/admin/config_badwords.lang.php

```diff
@@ -15,10 +15,10 @@
 
 $l['bad_word'] = "Slovo";
 $l['bad_word_desc'] = "Zadejte slovo, které si přejete, aby bylo filtrováno. Symbol '*' představuje libovolný počet znaků a symbol '+' představuje každý libovolný znak kromě mezery a nového řádku.";
-$l['bad_word_max'] = "Zakázáné slovo nemůže být delší než 100 znaků.";
+$l['bad_word_max'] = "Zakázané slovo nemůže být delší než 100 znaků.";
 $l['replacement'] = "Nahrazeno";
 $l['replacement_desc'] = "Zadejte řetězec, který nahradí filtrované slovo (pokud je pole prázdné, zobrazí se hvězdičky).";
-$l['regex'] = "Regular Expression";
+$l['regex'] = "Regulární Výraz";
```

Comment:
Tady bych možná řekl, že by to V mělo být s malým.

Reply:
Může být. Vzal jsem to podle původního textu, kde bylo obojí s velkým. Mám udělat druhý commit do pull requestu, nebo to squashnout do jednoho commitu?

Reply:
Za mě stačí fixup a pak squash, případně rovnou squashnout a force pushnout.